pkg/devfile/generator/utils_test.go

```diff
@@ -926,7 +1040,7 @@ func TestGetPortExposure(t *testing.T) {
 					Components: tt.containerComponents,
 				},
 			}
-			mapCreated := getPortExposure(devObj)
+			mapCreated, _ := getPortExposure(devObj, tt.filterOptions)
```

Comment:
the returned error should be checked

Reply:
true it should be checked. The reason i didnt because err will only be returned when using attributes and i found a bug with attributes in the api repo and doesnt return err properly but i will update this

Reply:
updated